src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,24 @@
 import React from 'react';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
-);
+import './UserInfo.scss';
+
+import { User } from '../../Types/user';
+
+type Props = {
+  user: User | null
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  if (!user) {
+    return null;
+  }
+
+  return (
+    <p>
+      {' Posted by '}
+      <a className="UserInfo" href={`mailto:${user.email}}`}>
```

Comment:
Use a more descriptive classname for the 'a' tag, such as 'user-info__email', to follow the BEM naming convention.